pkg/cli/admin/node/logs.go

```diff
@@ -113,6 +115,7 @@ func NewCmdLogs(baseName string, f kcmdutil.Factory, streams genericclioptions.I
 	cmd.Flags().BoolVar(&o.GrepCaseSensitive, "case-sensitive", o.GrepCaseSensitive, "Filters are case sensitive by default. Pass --case-sensitive=false to do a case insensitive filter.")
 	cmd.Flags().StringVar(&o.SinceTime, "since", o.SinceTime, "Return logs after a specific ISO timestamp or relative date. Only applies to node journal logs.")
 	cmd.Flags().StringVar(&o.UntilTime, "until", o.UntilTime, "Return logs before a specific ISO timestamp or relative date. Only applies to node journal logs.")
+	cmd.Flags().IntVar(&o.Boot, "boot", o.Boot, " Show messages from a specific boot, allowed [-100, 0]. Only applies to node journal logs.")
```

Comment:
Describe what happens if the boot doesn't exist.  Describe that you must use negative numbers (previous boots).